### PR TITLE
#999: Documentation for updating WordPress

### PR DESCRIPTION
**Pull request by Mte90**
As usual we can improve it later or during the review: https://github.com/Varying-Vagrant-Vagrants/VVV/issues/999

### CODE REVIEW

File: docs/en-US/references/update-site.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+category: 6. Reference
+order: 2
+title: Update WordPress websites
```
Comment (tomjn) [inline docs/en-US/references/update-site.md]: Updating WordPress Sites*, same with the URL

**Review by tomjn — APPROVED** [review]

**Comment by tomjn** [comment]
closes https://github.com/Varying-Vagrant-Vagrants/VVV/issues/999

File: docs/en-US/references/update-site.md [hunk 2]
```diff
@@ -1,28 +1,30 @@
...
+As well as reprovisioning, all the normal methods for updating work too, e.g.:
+
+ - Downloading WordPress from WordPress.org and extracting the zip file into `public_html`
+ - Using the built in updater in the Admin area
+ - Using WP CLI's core updater command
 
 ### custom-site-template-develop
 
-This provisioner use SVN or Git to download the latest trunk updated to the latest commit.  
-Natively this provisioner downloads the trunk version with SVN but is possible to convert it as Git version with the `develop_git` script.
```
Comment (Mte90) [inline docs/en-US/references/update-site.md]: I think that a mention about `develop_git` can be helpful.

**Comment by Mte90** [comment]
I was thinking to add a section:
```
### Execute only 1 provisioner at once

    vagrant provision --provision-with wordpress-one

This command will execute only the provisioner for the website specified, in this case `wordpress-one`.  
This solution sometimes is preffered instead of executing a provisioning for all the webistes available because is more fast.
```

**Comment by tomjn** [comment]
Until we have a `provision-site.sh` option for utilities, I don't want us encouraging `--provision-with`, otherwise new sites will provision without a TLS certificate being generated

**Pull request merged**